docs/en/sql-reference/functions/string-functions.md

```diff
@@ -4,6 +4,8 @@ sidebar_position: 170
 sidebar_label: Strings
 ---
 
+import VersionBadge from '@theme/badges/VersionBadge';
```

Comment:
It is a bit weird to show a minimum version for one function but not the 500 other functions. If we generate the web documentation from the in-source documentation, the result will be implicitly versioned.

(ping me if you like me to help with that, I worked on this topic last year)